test/framework/github.py

```diff
@@ -1317,20 +1318,37 @@ def test_github_create_test_report(self):
                 'log_file': logfile,
             }),
         ]
+        test_remove_name = 'REMOVEME'
+        test_remove_val = 'test-abc'
         init_session_state = {
             'easybuild_configuration': ['EASYBUILD_DEBUG=1'],
-            'environment': {'USER': 'test'},
+            'environment': {
+                'USER': 'test',
+                test_remove_name: test_remove_val,
+            },
             'module_list': [{'mod_name': 'test'}],
             'system_info': {'name': 'test'},
             'time': gmtime(0),
         }
+
+        # Test exclude_env_from_report_add/clear
+        exclude_env_from_report_add(test_remove_name.lower())  # Also check that the name is uppercased in the check
+
         res = create_test_report("just a test", ecs_with_res, init_session_state)
         patterns = [
             "**SUCCESS** _test.eb_",
             "**FAIL (build issue)** _fail.eb_",
             "01 Jan 1970 00:00:00",
             "EASYBUILD_DEBUG=1",
         ]
+        for pattern in patterns:
+            self.assertIn(pattern, res['full'])
+        self.assertNotIn(test_remove_name, res['full'])
+
+        exclude_env_from_report_clear()
+
+        res = create_test_report("just a test", ecs_with_res, init_session_state)
+        pattern.append(f"{test_remove_name} = {test_remove_val}")
         for pattern in patterns:
             self.assertIn(pattern, res['full'])
 
```

Comment:
Was this supposed to test `res['full']` again or was it supposed to test `res['overview']` ?